UI/js/controllers/controllers.js

```diff
@@ -101,3 +101,39 @@ dashboardApp.controller('termsController', ['Courses', 'Terms', '$rootScope', '$
   };
 
 }]);
+
+
+ /**
+  * Todo controller, using fake data for now
+  */
+
+dashboardApp.controller('todoController', ['ToDos', '$scope', function(ToDos, $scope) {
+  ToDos.getToDos('data/todolms/data-canvas-and-ctools.json').then(function(data) {
```

Comment:
From an outside perspective it may not be clear that these are specifically designed for testing. Perhaps a comment?